### PR TITLE
modular changes for Fedora 28

### PR DESCRIPTION
**Pull request by ausil**
Modular content will now live next to Everything including for
updates and updates-testing in Fedora 28. This "should" meeet
the needs.

Signed-off-by: Dennis Gilmore <dennis@ausil.us>

### CODE REVIEW

**Comment by ausil** [comment]
@adrianreber we are waiting on a compose to complete with the new content.

**Comment by adrianreber** [comment]
As soon as a compose is on the master mirror, I can test your patch in staging.

Any plans to remove the old modular content?

**Comment by ausil** [comment]
removing the old modular content now

File: mirrormanager2/lib/repomap.py [hunk 1]
```diff
@@ -97,47 +96,62 @@ def repo_prefix(path, category, ver):
...
-                        prefix = u'modular-fedora-server-f%s' % version
+                        prefix = u'rawhide-modular'
+                # fedora-modular-
+                elif isDebug:
+                    prefix = u'fedora-modular-debug-%s' % version
+                elif isSource:
+                    prefix = u'fedora-modular-source-%s' % version
+                else:
+                    prefix=u'fedora-modular-%s' % version
+            elif isUpdatesReleased:
```
Comment (ausil) [inline mirrormanager2/lib/repomap.py]: This entire portion is for the use case when we have a Everything tree inside of updates and updates-testing  which we will have next to Modular for updates and updates-testing

**Comment by adrianreber** [comment]
With the 0.8.4 release the new layout of the 'Modular' tree is detected correctly.

**Pull request closed without merging**